doc/codefmt.txt

```diff
@@ -82,7 +82,8 @@ Google's style. See https://github.com/mvdan/sh for details.
 Default: ['-i', '2', '-sr', '-ci'] `
 
                                                     *codefmt:shfmt_executable*
-The path to the shfmt executable.
+The path to the shfmt executable. String, list, or callable that takes no args
+and returns a string or a list.
```

Comment:
Looks like you manually edited this without updating the source in flags.vim it's generated from, so it'll get reverted next time someone regenerates the help docs.

Reply:
I've updated the flags.vim and checked that vimdoc generates the same string. I've also updated flags.vim to have the source for the changes I've done manually in https://github.com/google/vim-codefmt/pull/124. I can split it into another PR if it's preferred.

Reply:
Thanks! Same PR is fine for this, no worries.